src/background.js

```diff
@@ -2,25 +2,32 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
   const tabId = sender.tab.id;
   // console.log(message, tabId)
   let title = message;
-  let icon;
+  let icon = "h1";
 
   // h2 and hq is introduced from this post:
   // https://developers.google.com/web/updates/2017/12/chrome-loadtimes-deprecated
   // But the latest Chrome (68) seems use the value like this:
   // http/2+quic/43
   // Handle both cases for compatible
-  if (message === "h2") {
-    title = "HTTP/2";
-    icon = "h2";
-  } else if (message === "hq") {
+  if (message === "hq") {
     title = "HTTP/2 + QUIC";
     icon = "hq";
   } else if (message.startsWith("h3")) {
+    title = "HTTP/3";
     icon = "h3";
+  } else if (message.startsWith("h2")) { // "h2", "h2c"
```

Comment:
Thanks! Is there any documentation for the 'h2c' value? If yes then it would be great to add the doc link as a comment

Reply:
h2 is HTTP/2 over TLS
h2c is HTTP/2 over TCP
From [www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids)

Reply:
Added